### PR TITLE
Remove fakeOnStart workaround

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?
Remove fakeOnStart workaround which is not useful anymore since we don't reference Play.current global object anymore

## What is the value of this and can you measure success?
=> Play 2.5

### CODE REVIEW

**Review by jfsoul — APPROVED** [review]
🎉

**Comment by alexduf** [comment]
This is making me happy! :)

**Review by alexduf — APPROVED** [review]

File: common/app/services/DynamoDB.scala [hunk 1]
```diff
@@ -8,8 +8,8 @@ import conf.Configuration
 
 
 object DynamoDB {
-  private lazy val credentials = Configuration.aws.mandatoryCredentials
-  private lazy val region = Region.getRegion(Regions.fromName(Configuration.aws.region))
+  private val credentials = Configuration.aws.mandatoryCredentials
```
Comment (alexduf) [inline common/app/services/DynamoDB.scala]: You made all the properties lazy, except these two where you removed the laziness, is that voluntary?

Comment (TBonnin) [inline common/app/services/DynamoDB.scala]: It's an object so they don't need to be lazy right?

Comment (TBonnin) [inline common/app/services/DynamoDB.scala]: Reverted in 57aa80a7d3cb00dc713d7692db59c6729a5ab3cc

**Pull request merged**